### PR TITLE
Update principles to new words

### PR DESCRIPTION
**Pull request by edds**
This also removes all the examples so deleted a collection of the images
which are no longer used.

### CODE REVIEW

File: app/views/root/designprinciples.html.erb [hunk 1]
```diff
@@ -1,145 +1,68 @@
...
-    <p>
-    <a href="http://digital.cabinetoffice.gov.uk/2012/04/03/introducing-the-design-principles-alpha-for-gds/">ALPHA</a> <span>Last updated 2 July 2012</span>
-    </p>
-    </div>
-  </div>
-  <header>
+ <div id="wrapper" class="design-principles">
+  <header class="gds-principles">
     <h1>Government Digital Service <strong>Design Principles</strong></h1>
     <p>Listed below are our design principles and examples of how we’ve used them so far. These build on, and add to, our original <a href="http://www.flickr.com/photos/benterrett/7041509709/">7 digital principles</a>.</p>
```
Comment (tombye) [inline app/views/root/designprinciples.html.erb]: Léonie raised the point that this links to an inaccessible image on flickr, could we use this opportunity to change it?

Comment (edds) [inline app/views/root/designprinciples.html.erb]: It is a thing we should do yes, but I don't think it is worth holding up this change to make it.

File: app/views/root/designprinciples.html.erb [hunk 2]
```diff
@@ -1,145 +1,68 @@
...
 
- <div id="wrapper" class="design-principles"> 
-  <div class="beta-notice">
-    <div class="inner">
-    <p>
-    <a href="http://digital.cabinetoffice.gov.uk/2012/04/03/introducing-the-design-principles-alpha-for-gds/">ALPHA</a> <span>Last updated 2 July 2012</span>
-    </p>
-    </div>
-  </div>
-  <header>
```
Comment (jabley) [inline app/views/root/designprinciples.html.erb]: Does this mean it's no longer beta? :tada:

**Comment by tombye** [comment]
From a code perspective I'm only curious about one thing, shouldn't we change the heading structure so it's not all h1s?

Other than that it looks good to me but can you explain where the new words come from?

**Comment by tombye** [comment]
OK so corrections to the heading structure are like the image issue: out of the scope of this PR.

All else looks good to me :+1:

**Comment by benilovj** [comment]
The PR has obviously been merged already, but it would have been really nice to know *why* the words were changed... Neither the commit nor the PR message give any context.

**Pull request merged**